js_modules/dagster-ui/packages/ui-core/src/runs/RunAssetTags.tsx

```diff
@@ -12,6 +12,7 @@ export const RunAssetTags = (props: {run: RunFragment}) => {
   const queryResult = useQuery<RunAssetsQuery, RunAssetsQueryVariables>(RUN_ASSETS_QUERY, {
     variables: {runId: run.id},
     skip,
+    fetchPolicy: 'no-cache',
   });
```

Comment:
what UI surface does this power?

Reply:
Just the Run view

Reply:
hrmmm i guess?

At some point we should just make no-cache the default fetch policy - this query is entirely typical of the app and if it benefits from no caching, I don't see why we wouldn't do this in a lot more places.  I think we could invert all this and instead choose the 5-10 queries we DO want caching for?

Reply:
I think we should also just get the count here and fetch the full list on interaction while paginating it.

Reply:
But basically the crux of the issue is having all of these unbounded queries